airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -328,8 +342,14 @@ public KubePodProcess(ApiClient officialClient,
         .withMountPath(TERMINATION_DIR)
         .build();
 
-    Container init = getInit(usesStdin, List.of(pipeVolumeMount, configVolumeMount));
-    Container main = getMain(image, usesStdin, entrypoint, List.of(pipeVolumeMount, configVolumeMount, terminationVolumeMount), args);
+    Container init = getInit(usesStdin, List.of(pipeVolumeMount, configVolumeMount));;
```

Comment:
```suggestion
    Container init = getInit(usesStdin, List.of(pipeVolumeMount, configVolumeMount));
```

Reply:
In the future we may want to manually set the container requirements for the init / non-main containers. They should all be really lightweight.